src/summary.cpp

```diff
@@ -58,13 +59,13 @@ std::string pretty_print_bytes(std::size_t trace_size)
 
     std::array<std::string, 5> units = { { "B", "KiB", "MiB", "GiB", "TiB" } };
 
-    while(result_size > 1024)
+    while (result_size > 1024)
```

Comment:
The space character after `while` doesn't look like our clang-format style, did you use clang-format?

Reply:
Strange, I've used clang-format, and looking at my shells history, the same way I've used it to format all previous changes;